js-basics/react-with-arrays_users/src/components/Tag.js

```diff
@@ -1,5 +1,5 @@
 import "./Tag.css";
 
 export default function Tag({ tag }) {
-  return <li className="tag">{tag}</li>;
+  return <li className={tag === "admin" ? "tag--highlight" : "tag"}>{tag}</li>;
```

Comment:
`{tag === "admin" ? "tag--highlight" : "tag"}`

The code above is not returning the standard style `tag`, which should always be present.
A possible solution could be:
`{`tag ${tag === "admin" ? "tag--highlight" : ""}`}`